tests/phpunit/class-wc-stripe-checkout-sessions-ajax-handler-test.php

```
@@ -108,6 +114,248 @@ public function test_create_checkout_session(
+		WC()->cart->add_to_cart( $product->get_id(), 2 );
+
+		$captured_request = null;
+		$capture_body     = static function ( $request, $api ) use ( &$captured_request ) {
+			if ( 'checkout/sessions' === $api ) {
+				$captured_request = $request;
+			}
+			return $request;
+		};
+		add_filter( 'wc_stripe_request_body', $capture_body, 10, 2 );
+
+		$test_request = static function ( $return_value, $parsed_args, $url ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$parsed_args'. (undefined)

(UnusedFormalParameter)